app/assets/javascripts/tylium.js

```diff
@@ -33,7 +33,7 @@
 //= require tylium/engines
 //= require tylium/keyboard_shortcuts
 
-//= require tylium/modules/auto_save
+//= require tylium/modules/local_auto_save
```

Comment:
Please see: https://github.com/dradis/dradis-ce/pull/656#pullrequestreview-404963880

I think that moving this to `/auto_save/local` would give us better structure once both PRs are merged.

Reply:
Done!